[Docs] Sylius Cloud by Platform.sh (#16279)
Sylius/Sylius: docs/cookbook/deployment/introduction-to-sylius-cloud.rst

Introduction to Sylius Cloud by Platform.sh
===========================================

> Sylius Cloud by Platform.sh is a way of efficient and scalable Sylius hosting. You can deploy and manage your application

Reviewer: ```suggestion
Sylius Cloud by Platform.sh provides an efficient and scalable solution for hosting Sylius. You can deploy and manage your application
```

- \- **Build process** - the series of steps that are executed to prepare your application for deployment to a specific environment. These steps involve installing dependencies (Composer), warming-up application cache, building NodeJS assets, and performing any other tasks necessary to ensure that your application is ready to run in the target environment.
- \- **Deployment Process** - the process of replacing the existing application with the already built, updated version. The current application version is hosted until the whole deployment process exits without an error.
- \- **Relations** - the connections established between different services or components within your project. These connections enable communication and interaction between services, allowing them to work together seamlessly to support your application.

How to deploy Sylius to Sylius Cloud by Platform.sh?
====================================================

.. tip::

>     Start with reading `Platform.sh documentation <https://docs.platform.sh/frameworks/symfony.html>`_.

Reviewer 2: I would also like to point out that in the `Prepare a Platform.sh project` section, after creating the project, there is an instruction of starting and deploy, where you can go step by step with the correct IDs, etc., of your newly created project. You might consider including a screenshot or something similar to illustrate this. Check it out:

![Screenshot 2024-05-21 at 10 29 18](https://github.com/Sylius/Sylius/assets/53942444/aad0eb51-a96a-4241-833f-b09f660ec0fb)

Reviewer 2: it is a page of the `test` organisation with the `custome project 1` project and standard-with-113 environment that is a main production branch from the https://github.com/Sylius/Sylius/pull/16279/files#diff-7a7f6077d161339cde0de18ff4a9460e131db5bcb33968927327e194fd082248 step

- \- `.env.platform.dist` - The environment file automatically used by the application during the build process.
- \- `.platform/routes.yaml` - This file defines the routes and how traffic is routed to the application. It specifies the incoming URLs and how they should be directed to the appropriate services or applications. Routes can be configured to handle different paths, domains, or other conditions.
- \- `.platform/services.yaml` - This file is used to define additional services or dependencies required by the application. It allows defining various types of services such as databases, caches, search engines, etc., along with their configuration. Services defined in this file are automatically provisioned and integrated with the application environment.

1. Prepare a Platform.sh project
--------------------------------

* Create an account on `Platform.sh <https://platform.sh/>`_.

> * Create a new project, name it (**MyFirstShop** for example) and select the **Blank project** template.

Reviewer 2: We also need to create an organisation, then we will be asked to provide phone number for the sms code to authorize, if we haven't done it earlier, for someone it could be blocking, I would be after mentioning it before creating project like "if it's your first time with platform.sh you may be asked to create your organisation, do it, then you will be asked to provide phone number, for the sms code authorization, don't worry it is not stored, it is just for the security reasons" or something like that

Introduction to Sylius Cloud by Platform.sh
===========================================

> Sylius Cloud by Platform.sh is provides an efficient and scalable for hosting Sylius. You can deploy and manage your application

Reviewer: ```suggestion
Sylius Cloud by Platform.sh provides an efficient and scalable solution for hosting Sylius. You can deploy and manage your application
```
